Add conclusions & small content suggestions (#135)
DE-RSE/2023_paper-RSE-groups: paper.tex

This will improve the chances of a project being successfully completed in a timely manner.
That means a central RSE unit has more RSE competencies than any individual research group in the institution.
This allows members of that unit to bring in new ideas or transfer them from other collaborations to these groups.

> Furthermore, by centralizing RSE expertise, the unit benefits from economies of scale, making it feasible to handle also less frequently needed tasks across multiple groups.

Reviewer: ```suggestion
Furthermore, by centralising RSE expertise, the unit benefits from economies of scale, making it feasible to handle also less frequently needed tasks across multiple groups.
```

BE spelling

Author: Thanks & sorry for the overhead!

If the research within an institution heavily relies on specific pieces of software,
it is of vital importance for the long term success of the institution to sustain these pieces of software.

> Relying on a workforce that is subject to academic labour turnover poses the risk of knowledge loss and incurs a significant overhead due to the need to constantly retrain new staff.

Reviewer 2: ```suggestion
Relying on a workforce that is subject to academic labour turnover poses the risk of knowledge loss and incurs a significant overhead due to the need to constantly search for and then retrain new staff.
```

Author: Makes sense, thanks!

Moreover, data science curricula already exist and more are in the process of being created.
A curated and continuously updated list of these programs is available at~\cite{learnandteachlearn}.

\section{Conclusions}
\label{sec:conclusions}

> In this paper, we have outlined a vision for central RSE units in research institutions,  focusing particularly on the German research landscape.

Reviewer 2: ```suggestion
In this paper, we have outlined a vision for central RSE units in research institutions, focusing particularly on the German research landscape.
```

We have proposed a modular framework for structuring the services of such units, identifying nine fundamental modules and providing concrete pathways for their implementation.
Our survey of existing RSE units illustrates that such structures can take various forms, ranging from consultation-focused groups to those offering extensive software development services.
Yet, all serve to consolidate expertise within a research institution, strengthen cooperation across scientific disciplines, and increase the overall quality of research.

Besides addressing local needs in software development and maintenance, well-established RSE units also bring broader institutional benefits, such as improved funding acquisition or enhanced reputational standing.
They further enable closer collaboration among related domains like HPC, data management, and digital infrastructure.
This ultimately leads to more sustainable, reproducible, and impactful research on and with software-based methods.

> We encourage decision makers to strategically invest in and formalize central RSE units, recognizing their role as indispensable partners in modern research environments.

Reviewer: ```suggestion
We encourage decision makers to strategically invest in and formalise central RSE units, recognising their role as indispensable partners in modern research environments.
```
BE spelling

Author: Thanks & sorry for the overhead!
